cirq-google/cirq_google/devices/google_noise_properties.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Class for representing noise on a Google device."""
+
+from dataclasses import dataclass, field
+from typing import Dict, List, Set, Type
+import numpy as np
+
+import cirq, cirq_google
+from cirq import _compat, devices
+from cirq.devices import noise_utils
+from cirq.transformers.heuristic_decompositions import gate_tabulation_math_utils
+
+
+SINGLE_QUBIT_GATES: Set[Type['cirq.Gate']] = {
+    cirq.ZPowGate,
+    cirq.PhasedXZGate,
+    cirq.MeasurementGate,
+    cirq.ResetChannel,
+}
+SYMMETRIC_TWO_QUBIT_GATES: Set[Type['cirq.Gate']] = {
+    cirq_google.SycamoreGate,
+    cirq.FSimGate,
+    cirq.PhasedFSimGate,
+    cirq.ISwapPowGate,
+    cirq.CZPowGate,
+}
+ASYMMETRIC_TWO_QUBIT_GATES: Set[Type['cirq.Gate']] = set()
+
+
+@dataclass
+class GoogleNoiseProperties(devices.SuperconductingQubitsNoiseProperties):
+    """Noise-defining properties for a Google device.
+
+    Inherited args:
```

Comment:
It seems strange to have a copy of the same documentation in two locations. Is this standard practice? Perhaps you should instead reference the docstring in `devices.SuperconductingQubitsNoiseProperties`, and only discuss the difference between this class and its parent.

Reply:
In most cases it's OK to omit parent-class documentation, but I wanted to include it here since both this and the parent are `@dataclass`es. As a result, the source of the "inherited args" is somewhat opaque - documenting them here can save the reader the work of inspecting the parent class.